source/community/reactnative/package.json

```diff
@@ -53,8 +54,9 @@
     "adaptivecards-templating": "1.0.0-rc.0",
     "antlr4ts": "^0.5.0-alpha.3",
     "assert": "^2.0.0",
-    "react-native-video": "^4.4.2",
-    "react-native-keyboard-aware-scroll-view": "^0.9.5"
+    "patch-package": "^6.4.7",
```

Comment:
should `patch-package` be under dependencies or devDependencies?

Reply:
Updated